src/api/books.tsx

```diff
@@ -0,0 +1,9 @@
+import { googleBooksApi } from "./googleApi";
+
+export const getBooks = (word) => {
+  return googleBooksApi.get("volumes?q=" + word + "&maxResults=30");
+};
+
+export const getBookInfo = (id) => {
+  return googleBooksApi.get("volumes/" + id);
```

Comment:
you can use template strings 

```ts
`volumes/${id}`
```

Reply:
You need to take advantage to TypeScript and add types for the data returned by these APIs to make your projects type safe

you can do this with Axios like this

`googleBooksApi.get<Book>("volumes/" + id);`

and you define the `Book` type in your `types` folder

```ts
export interface Book {
id: string
...etc
}
```